### PR TITLE
[2469] Document Document Locking Issue With Office Online and Chrome 

### LINKED ISSUE #2469: WOPI: Document Chrome Document Locking
We found out that when you use chrome and edit a document via ownCloud in Office Online - the lock is not released with the closing of the document, but after the 30 min time out or a manual release.

This needs to be documented.

Probably after this part:

https://doc.owncloud.com/server/admin_manual/enterprise/collaboration/msoffice-wopi-integration.html#lock-timeout

Maybe as "Known Issues"

### PR DESCRIPTION
**Pull request by settermjd**
This fixes #2469, documenting the document lock issue when using Chrome to edit Office Online documents.

### CODE REVIEW

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 1]
```diff
@@ -67,3 +67,10 @@ You can always click on the lock icon next to your file name and unlock it manua
 
 If a user is editing a file and loses their internet connection the lock will timeout, freeing the lock, after 30 minutes. 
 Please refer to {wopi-timeout-documentation-url}[the WOPI documentation] for further information.
+
+== Known Issues
+
+=== Document Locks Are Not Released When Using Google Chrome
+
+When editing a document with Google Chrome (and Chromium) via ownCloud in Office Online, the document lock is _not released_ when the document is closed.
+The document lock is only released after the 30-minute timeout or a manual lock release.
```
Comment (phil-davis) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc]: Note: this is actually a Microsoft bug. So should we mention that? - and that therefore the bug might go away (or get worse ;) in the future out of the control of ownCloud.

Comment (phil-davis) [reply modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc]: And could specifically add some thing like:
```
In this case, try to remember to manually unlock the document before closing.
```

Comment (settermjd) [reply modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc]: > Note: this is actually a Microsoft bug. So should we mention that? - and that therefore the bug might go away (or get worse ;) in the future out of the control of ownCloud.

It's definitely worth mentioning, as that makes it clear to the reader that it's not ownCloud's fault nor responsibility. Do you have a link to the issue? If not, I'll search for one.

Comment (settermjd) [reply modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc]: > In this case, try to remember to manually unlock the document before closing.

I'll work that in to this PR.

Comment (settermjd) [reply modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc]: @phil-davis, I've added in two links and [the workaround](https://github.com/owncloud/docs/pull/2474#discussion_r394865058) that you mentioned.

**Comment by phil-davis** [comment]
@settermjd please make backports 
I'm pretty sure this will be an issue regardless of if ownCloud core is 10.3.* or 10.4.* - so backport to both docs branches.

**Pull request merged**